Kitodo/src/test/java/org/kitodo/services/data/BatchServiceIT.java

```diff
@@ -36,8 +37,9 @@ public static void prepareDatabase() throws DAOException, IOException, ResponseE
     }
 
     @AfterClass
-    public static void cleanDatabase() {
+    public static void cleanDatabaseAndIndex() throws IOException, ResponseException{
```

Comment:
It is preparation for upcoming. This pull request can wait for next ones which will use this functionality.